File: frontend-appstore/PusulaService/Models/AdminNotificationModels.swift
import Foundation

struct AdminNotificationDTO: Codable, Identifiable {
    let id: Int
    let title: String
    let message: String
    let severity: String
    let category: String
    let referenceType: String?
    let referenceId: Int?
    let read: Bool
    let createdAt: String
}

struct NotificationUnreadCountDTO: Codable {
    let count: Int
}

File: frontend-playstore/PusulaService/app/src/main/java/com/pusula/service/data/model/AdminNotificationModels.kt
package com.pusula.service.data.model

data class AdminNotificationDTO(
    val id: Long,
    val title: String,
    val message: String,
    val severity: String,
    val category: String,
    val referenceType: String? = null,
    val referenceId: Long? = null,
    val read: Boolean = false,
    val createdAt: String
)

data class NotificationUnreadCountDTO(val count: Long = 0)
